### PR TITLE
Spelling

### PR DESCRIPTION
**Pull request by mhagdorn**
change spelling to British English and replace some American English terms with BE ones

It might be an idea to port the spell check action  https://github.com/the-teachingRSE-project/competencies/pull/148 divised by @BeastyBlacksmith

### CODE REVIEW

File: paper.tex [hunk 1]
```diff
@@ -148,8 +148,8 @@ \subsubsection{Existing RSE efforts}
 The concept of central RSE teams is also not untested.
 Examples \todo{FLO+PMS: Do we want to mention all? DLR has something similar. So far it’s only the authors’ group.} of organisational RSE teams in Germany are
  the Helmholtz HIFIS group\footnote{\url{https://events.hifis.net/category/4/}}\autocite{haupt_hifis_consulting_2021},
- the Scientific Software Center in Heidelberg\footnote{\url{https://www.ssc.uni-heidelberg.de/en}}\autocite{ulusoy_heidelberg_ssc_2024},
- the Competence Center Digital Research (zedif) in Jena\footnote{\url{https://www.zedif.uni-jena.de/en/}},
+ the Scientific Software Centre in Heidelberg\footnote{\url{https://www.ssc.uni-heidelberg.de/en}}\autocite{ulusoy_heidelberg_ssc_2024},
+ the Competence Centre Digital Research (zedif) in Jena\footnote{\url{https://www.zedif.uni-jena.de/en/}},
```
[the remark this answers is not in the document]
Comment (dokempf) [reply paper.tex:152]: Yes, we (the SSC) are explicitly a centER. That was a deliberate decision.

[the remark this answers is not in the document]
Comment (mhagdorn) [reply paper.tex]: you are quite right, I checked the spelling of the capital Centres/Centers and I think I now have the right spellings

**Review by jngrad — APPROVED** [review]

**Comment by BeastyBlacksmith** [comment]
> It might be an idea to port the spell check action [the-teachingRSE-project/competencies#148](https://github.com/the-teachingRSE-project/competencies/pull/148) divised by @BeastyBlacksmith

Let me know, if I can help with that

**Comment by mhagdorn** [comment]
I have conferred with a British colleague, on- and  offboarding are indeed terms that are used in British English. Clearly some new-fangled HR speak that I hadn't come across before.

I cannot merge the request, so someone who can please go ahead.

**Pull request merged**